docs/releases/2.2.txt

```diff
@@ -41,6 +41,9 @@ database constraints. Constraints are added to models using the
 Minor features
 --------------
 
+* Allowed override formset order field widget.
```

Comment:
This message should go under the "Forms" heading. And here's slightly better wording for this.

Allow overriding formset order field widget.

Reply:
Thank you. done